Automattic/sensei
# Fix: Students page now will show all courses enrolled even if it's more than 10.

Fixes #5336

### Changes proposed in this Pull Request

Previously, when you loaded the Students page, it would query to get up to 10 courses enrolled by students.  It would show 3, then hide up to 7 which would appear when you clicked on a "more" button.  

The problem with this is that
- If you had more than 10 courses on a student, it would not show them
- It could be a performance issue if you had a page of 20 students all with 10 courses each, and to show all courses could be worse.

What this PR does is
- Moves the "more" button to be an AJAX call that fetches the rest of the courses for the student you are wanting. 
- This way even if a student is enrolled to 100 courses, it happens asynchronously and only one student at a time.

### Testing instructions

* Enable this PR
* Make sure you have at least 11 courses published
* Make sure you have a few students, testing with > 2 is good
* Go to the Students page
* Add all the courses to all the students by clicking the `...` menu like this:
<img width="399" alt="Screen Shot 2022-10-07 at 1 32 33 PM" src="https://user-images.githubusercontent.com/3220162/194648140-6107ee18-2db9-49c5-a6d6-d87599a8e99c.png">
* Try clicking the "+X More" button
* Make sure all the right courses show up for each student.

### Screenshot / Video
![ajax-get-courses3](https://user-images.githubusercontent.com/3220162/194648280-fe7da3bd-aaee-4fe9-a8e7-2ba298f8b839.gif)
